backend/clubs/views.py

```diff
@@ -2929,6 +2931,18 @@ def destroy(self, request, *args, **kwargs):
                 f"If you would like to do this, email {settings.FROM_EMAIL}."
             )
 
+        if (
+            Ticket.objects.filter(event=event)
+            .filter(Q(owner__isnull=False) | Q(holder__isnull=False))
+            .exists()
```

Comment:
Was this change made to enable  admin to delete event even though it isn't over? The original code checks `now <= event.end_time`

Reply:
Yeah -- if a ticketed event is cancelled & nobody's bought tickets for it, it should be deletable.